Sources/LiveViewNativeCharts/Modifiers/View Fundamentals Modifiers/ChartBackgroundModifier.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  ChartBackgroundModifier.swift
+//
+//
+//  Created by murtza on 22/06/2023.
+//
+
+import Charts
+import SwiftUI
+import LiveViewNative
+
+/// Adds a background to a view that contains a chart.
+///
+/// ```html
+/// <Chart template={:content} modifiers={chart_background(@native, alignment: :center)}>
+///   <chart_background:background>
+///     <Color name="system-green" />
+///   </chart_background:background>
```

Comment:
This uses the old syntax for templates and modifiers. Now it should be:
```suggestion
/// <Chart modifiers={chart_background(alignment: :center, content: :background)}>
///   <Color name="system-green" template={:background} />
```